src/Features/CSharp/Portable/InitializeParameter/InitializeParameterHelpers.cs

```diff
@@ -33,6 +33,16 @@ public static SyntaxNode GetBody(SyntaxNode functionDeclaration)
             }
         }
 
+        public static ImmutableArray<SyntaxNode> GetParameters(SyntaxNode node, SyntaxGenerator generator)
+        {
+            if (node is SimpleLambdaExpressionSyntax simpleLambda)
+            {
+                return ImmutableArray.Create(simpleLambda.Parameter as SyntaxNode);
+            }
+
+            return generator.GetParameters(node).ToImmutableArray();
```

Comment:
FYI, once https://github.com/dotnet/roslyn/pull/37922 goes in, you'll be able to remove all of this code.

Reply:
We cherry picked the change over from 16.4-p1 branch